contracts/uos.system/include/eosio.system/eosio.system.hpp

```diff
@@ -217,61 +297,186 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( voter_info, (owner)(proxy)(producers)(staked)(last_vote_weight)(proxied_vote_weight)(is_proxy)(flags1)(reserved2)(reserved3) )
    };
 
-    struct [[eosio::table, eosio::contract("uos.system")]] calc_voter_info {
-        name                owner;       /// the voter
-        std::vector<name>   calculators;     /// the calculators approved by this voter
+   /**
+    * Calculator Voter info.
+    *
+    * @details Stores information about the voter related to voting for Calculators:
+    * - `owner` the voter
+    * - `calculators` the calculators approved by this voter
+    */
+   struct calc_voter_info {
+      name                owner;
+      std::vector<name>   calculators;
 
-        uint64_t            primary_key()const { return owner.value; }
+      uint64_t primary_key()const { return owner.value; }
 
-        // explicit serialization macro is not necessary, used here only to improve compilation time
-        EOSLIB_SERIALIZE( calc_voter_info, (owner)(calculators) )
-    };
+      // explicit serialization macro is not necessary, used here only to improve compilation time
+      EOSLIB_SERIALIZE( calc_voter_info, (owner)(calculators) )
+   };
 
-    struct [[eosio::table, eosio::contract("uos.system")]] voter_rates {
-        name                        owner;         /// the voter
-        double                      social_rate = 0;   /// voter's share in the total social activity, a fraction between 0 and 1
-        double                      transfer_rate = 0; /// voter's share in the total transfer activity, a fraction between 0 and 1
+   /**
+    * Voter rates.
+    *
+    * @details stores information on uos calculated rates:
+    * - `owner` the voter
+    * - `social_rate` voter's share in the total social activity, a fraction between 0 and 1
+    * - `transfer_rate` voter's share in the total transfer activity, a fraction between 0 and 1
+    */
+    struct voter_rates {
```

Comment:
changed from `struct [[eosio::table, eosio::contract("uos.system")]] voter_rates {` to `struct voter_rates {`